### PR TITLE
Update README to describe goals, s/best/working/

### LINKED ISSUE #28: Clarify process goals
* To provide a catalog current working practices, which are not necessarily *best* practices, just what we do
* To gauge consensus on those working practices, not to argue about whether something is best or not, just whether they're reasonable
* To provide a starting point and default set of expectations, not to mandate or set in stone
* To provide discussion topics, not new hire job requirements
* To provide a set of guides for new hires to learn from, not absolute requirements they must adopt || die()

* The process is broad, imperfect and fast at the outset.  Future discussions can refine, correct and go deep.
* The process can be modified as needed
* Non-substantive changes do not have a waiting period once 4 :+1:s are gathered (e.g. updating an index of links versus creating new content)

[Wording is for sentiment, not final]

### PR DESCRIPTION
**Pull request by azaroth42**
Add goals, change language from "best" to "working" practices. Closes #28.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,25 @@ waiting a week for everybody to leave feedback**.
 
 [contribution guidelines]: /CONTRIBUTING.md
 
+Goals
+-----
+
+The goals of the building the Developer Playbook are to:
```
Comment (darrenleeweber) [inline README.md]: change `the building` to `building`.

**Comment by darrenleeweber** [comment]
:+1: for more explicit identification of goals and expectations.  However, does this also entail the subdirectory `best_practices` should be renamed?

File: README.md (hunk 1 above)
Comment (azaroth42) [inline README.md]: Thanks! Fixing...

**Comment by azaroth42** [comment]
I was going to do a second PR for the structure change, as the structure has been controversial.

**Comment by cbeer** [comment]
-0. Our working practices **are** our best practices, and when there are better practices in the world (that work for the group), we should try to adopt them as our own.

**Comment by darrenleeweber** [comment]
@cbeer - so let's consider python!  Clearly python is a more efficient language than ruby, with an ever growing ecosystem in a diverse range of applications and widely used in major tech firms (e.g. google), and so it is clearly a best practice to use python, so DLSS should switch from ruby?  I doubt this would fly in DLSS because there is substantial tech-debt and dev-skills in ruby.  The 'best' practice may be out there, but it's not necessarily in DLSS and any implication that DLSS is using the 'best' practice entails substantial argument both for and against not only DLSS, but also other practices.  That's not a goal of this play book.  Document 'working' practices that might aspire to be 'best' practices or at least refer to arguments or debate about the issue, but don't be so bold as to claim 'best' practices without justifications.

**Comment by cbeer** [comment]
@darrenleeweber I think we disagree on the scoping here. I'm not arguing that our best practices are globally recognized best practices (if such a thing exists..  I'm sure you could incite many a flame war trying to argue that). I'm just arguing that our working practices are our best practices. I have a vague feeling that saying "best practices" is actually important here and possibly mediates several kinds of conflict:

- in the case of conflicting working practices, where one practice is acknowledged as the "correct" departmental approach, we should choose the "correct" one, and not document that e..g "local practice until 2007 within the webteam was to use xyz"

- for newcomers, I'm worried that "working practices" may not be strong enough, and could elicit a response like "yeah, sure, that's their working practice, but I'm going to use a better practice from X"

- I hope that by calling them best practices, when we contribute to this documentation it gives us pause when our local practices diverge from best practices in our wider communities, and possibly hint at why this is so, or give us some motivation to form consensus within our group around new practices. Similarly, when better practices emerge, we want them to challenge our existing practices, not merely be discarded as "not our practices"

Note, too, that I voted `-0` and can live with the change (but clearly don't support it). It's only a word, and can be compensated for through increased vigilance when authoring and explaining this guide.

**Comment by azaroth42** [comment]
Will change the wording in the PR to try and reflect "current local best practices".

**Comment by cbeer** [comment]
+0, thanks.

**Comment by ndushay** [comment]
:+1: this works for me, for now.  As the playbook grows, we may well want to revisit this text, but I think it is an improvement over what exists without this PR.

**Comment by darrenleeweber** [comment]
I would like to see part of the goals note that this playbook must include links to existing code projects that are or will be exemplars of the local best practices.  This should help to provide concrete, working code examples and ensure that this playbook is tied to DLSS projects that actually implement the practices.

**Comment by azaroth42** [comment]
Lets make a new issue for that :)

**Comment by ndushay** [comment]
@darrenleeweber I agree that ultimately the playbook should be example rich;  we are just getting started, and I'm confident the top level readmes will evolve as we go..

**Comment by drh-stanford** [comment]
:+1:

**Comment by mjgiarlo** [comment]
Needs rebase, but :+1:

**Comment by mjgiarlo** [comment]
Also related to the emerging (draft) HyBox dev practices: 
[SoftwareDevelopmentPractices.pdf](https://github.com/sul-dlss/DeveloperPlaybook/files/32170/SoftwareDevelopmentPractices.pdf)

**Comment by mjgiarlo** [comment]
@azaroth42 inclined to rebase this so it can be merged?

**Comment by mjgiarlo** [comment]
Taken up again in #52. Closing. (Thanks, @azaroth42!)

**Pull request closed without merging**